config/betterquesting/DefaultQuests/Quests/Tier0StoneAge-AAAAAAAAAAAAAAAAAAAAAQ==/GettingCharcoal-AAAAAAAAAAAAAAAAAAAAGA==.json

```diff
@@ -35,7 +35,7 @@
       "lockedProgress:1": 1,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "It\u0027s time to get yourself some charcoal. For that, we\u0027re going to build a coke oven. Good that you just collected a lot of resources! (What a coincidence...)\n\n§3Even though Waila says you can\u0027t, you can punch these blocks to get them back if you mess up the placement before you have a metal pickaxe."
+      "desc:8": "It\u0027s time to get yourself some charcoal. For that, we\u0027re going to build a coke oven. Good that you just collected a lot of resources! What a coincidence...\n\nEven though Waila says you can\u0027t, you can punch these blocks to get them back if you mess up the placement before you have a metal pickaxe.\n\nIt may be useful to remember that the oven doesn\u0027t automatically output its fluid contents - for that, you will need to apply a pump to it in LV, which is normally called a pump cover."
```

Comment:
no you dont need to wait for LV pumps thats just incorrect

Reply:
Addressed in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/5afb3492d9a85bef1a82e406a9a09827a4e628ac. Mentioned faucets and pumps as an option.